test/test_images/emptydir/emptydir.go

```diff
@@ -18,37 +18,36 @@ package main
 
 import (
 	"flag"
-	"fmt"
 	"log"
 	"net/http"
 	"os"
+	"path"
 
 	"knative.dev/serving/test"
 )
 
-var path string
+func testfilePath() string {
+	base := os.Getenv("DATA_PATH")
+	if base == "" {
+		base = "/data"
+	}
+	return path.Join(base, "testfile")
+}
 
 // Add content to a file in the emptyDir volume
 func init() {
```

Comment:
Do we actually need this to be in an init any more? It's not initialising any global variables now so I think we could move it in to main

Reply:
Fair, changed.